akka-stream-typed/build.sbt

```diff
@@ -0,0 +1,8 @@
+import akka.{ AkkaBuild, Formatting }
+
+AkkaBuild.defaultSettings
+AkkaBuild.mayChangeSettings
+Formatting.formatSettings
+
+disablePlugins(MimaPlugin)
```

Comment:
let's put everything in akka-typed for now, until we do the proper split

Reply:
ok, will change :)